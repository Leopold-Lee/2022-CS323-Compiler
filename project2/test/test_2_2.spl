int foo()
{
  int a = 10;
  a = bar();
  return 0;
}

int bar() {
    float d = 0;
    return 0;
}

int bar(int b) {
    b = b + 1;
    return b;
}