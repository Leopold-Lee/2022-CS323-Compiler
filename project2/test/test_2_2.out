Error type 2 at Line 4: undefined function: bar
Error type 4 at Line 13: redefine function: bar